public/index.hbs

```diff
@@ -46,6 +46,7 @@
 
         {{> '../src/templates/dictionary/striped'  }}
       </div>
+    </div>
```

Comment:
This adds an extra closing `div`. Is this where we want it? If so, we need to remove the extra from line 114.